packages/puppeteer-core/src/api/Page.ts

```diff
@@ -3174,6 +3175,11 @@ export abstract class Page extends EventEmitter<PageEvents> {
    * method is only available in Chrome.
    */
   abstract openDevTools(): Promise<Page>;
+
+  /**
+   * {@inheritDoc BluetoothEmulation}
+   */
+  abstract get bluetoothEmulation(): BluetoothEmulation;
```

Comment:
I would just call it `bluetooth` for brevity.

Reply:
done